### DISCUSSION TITLE
Help with WCA auth

### DISCUSSION THREAD

**Discussion by Sosna123** [cs0x7f/cstimer#412] [General]
I'm trying to make my own timer app and wanted to make login through wca, and everytime i fetch i get 401 not authorized. Can someone tell me what am I doing wrong? (scope is default; jscookie.get returns the bearer token as a string)

![obraz](https://github.com/user-attachments/assets/0e36adbe-7d77-4dbe-9580-e34cef56c585)
![obraz](https://github.com/user-attachments/assets/1805e55c-da18-45bf-8fd3-41c722d6d388)
![obraz](https://github.com/user-attachments/assets/56dc77b5-b99b-4a8a-8466-5062fba562e4)

**Comment by cs0x7f** [comment]
I suggest you ask WST (software@worldcubeassociation.org) for help.
